pages/host.dashboard.js

```diff
@@ -194,7 +193,7 @@ class HostDashboardPage extends React.Component {
                   <FormattedMessage id="HostedCollectives" defaultMessage="Hosted Collectives" />
                 </LinkContainer>
               </Link>
-              {parseToBoolean(getEnvVar('HOST_DASHBOARD_REPORTS')) && (
+              {parseToBoolean(process.env.HOST_DASHBOARD_REPORTS) && (
```

Comment:
Why?

Reply:
@Betree : I was actually going to revisit this one; but forgot. 🤯 Somehow `getEnvVar` doesn't seem to take the environment variables from `.env` file and always return `false` for me. I think I have seen this issue before as well but cannot remember where we saw this. Anyways, this is the reason why I changed it; `process.env` always works for me without issues and somehow I am more inclined to use that. But I've changed it back for now until I figure out exactly what's the reason behind this. 🤔